tests/test_bridge.py

```diff
@@ -95,6 +95,18 @@ async def test_authenticate_no_credentials() -> None:
             await box._ensure_token()
 
 
+@respx.mock
+@pytest.mark.asyncio
+async def test_authenticate_no_credentials_url() -> None:
+    """It exits with a status code of zero."""
+    async with httpx.AsyncClient() as client:
+        box = SFRBox(ip="https://sfrbox.example.com", client=client)
+        with pytest.raises(
+            SFRBoxAuthenticationError, match="Credentials not set"
+        ):
+            await box._ensure_token()
+
+
```

Comment:
Indeed this is not testing anything...
You need to duplicate a test that makes use of `respx`

Then you can check that the `ip` argument get correctly translated to the `respx.get` prefix

You can duplicate `test_authenticate_method_not_allowed`, but you can't duplicate `test_authenticate_no_credentials` as that doesn't even try to connect

Reply:
There it is. It took me way too long to understand the mocking mechanism :sweat_smile: